src/react-demo/Icons.es6.js

```diff
@@ -712,6 +712,10 @@ const ICONS = [
     name: 'Zap',
     component: Zap
   },
```

Comment:
If this screenshot is what you are seeing after using it in product! lgtm!!

![image](https://user-images.githubusercontent.com/3481095/175382024-681dc74b-731d-47d8-a42b-b589ef9c685f.png)